_inc/client/plans/accented-card.jsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const AccentedCard = props => {
+	const { header, body } = props.children;
+
+	return (
+		<div className="accented-card">
+			<div className={ 'accented-card__header' }>{ header }</div>
```

Comment:
You could just use HTML's `<header>` and `<section>` elements in order to style these without doing the props trick above or having to use custom elements like `<AccentedCardHeader>`